source/MaterialXContrib/MaterialXNode/MaterialXNode.cpp

```diff
@@ -89,28 +91,20 @@ MStatus MaterialXNode::initialize()
 
 void MaterialXNode::createOutputAttr(MDGModifier& mdgModifier)
 {
-	if (materialXData && materialXData->isValidOutput())
-	{
-        const MaterialX::StringMap& outputMap = materialXData->getPathOutputMap();
-        if (outputMap.size())
-        {
-            MString outputName(outputMap.begin()->second.c_str());
-            if (outputName.length())
-            {
-                MFnNumericAttribute nAttr;
-                _outAttr = nAttr.createColor(outputName, outputName);
-                CHECK_MSTATUS(nAttr.setStorable(false));
-                CHECK_MSTATUS(nAttr.setInternal(false));
-                CHECK_MSTATUS(nAttr.setReadable(true));
-                CHECK_MSTATUS(nAttr.setWritable(false));
-                CHECK_MSTATUS(nAttr.setCached(true));
-                CHECK_MSTATUS(nAttr.setHidden(false));
-
-                mdgModifier.addAttribute(thisMObject(), _outAttr);
-                //		CHECK_MSTATUS(addAttribute(_outAttr));
-            }
-        }
-	}
+    if (materialXData && materialXData->isValidOutput())
+    {
+        const MString outputName(MaterialX::OgsXmlGenerator::OUTPUT_NAME.c_str());
```

Comment:
We hard-code the output name now so we don't need to worry about name changes. We always promote to float3 for type consistency, but will need to look at float4 output afterwards so we can get transparency working.